src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -307,6 +307,22 @@ public function testDeprecatedAlias()
         $builder->get('foobar');
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Since symfony/dependency-injection 5.1: The "foobar" service alias is deprecated. You should stop using it, as it will be removed in the future.
+     */
+    public function testDeprecatedAliasWithPackageAndVersion()
```

Comment:
what about deprecated definition with package and version too ?

Reply:
Added